distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -79,3 +79,17 @@ grant codeBase "file:/pax-logging-log4j2" {
     //  permission java.io.FilePermission "/the/new/log/directory/-", "read, write";
     //  permission java.io.FilePermission "/the/new/log/directory", "read, write";
 }
+
+//                       SSH Permissions
+//  Adding required permissions for commands executed via SSH.
+//
+//  Access to files outside of the user or karaf home directories is restricted when using SSH. To allow a command to access a directory
+//  when using SSH, permission entries need to be added in the following field.
+//
+grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.karaf.sshd.core" {
```

Comment:
❓ I am surprised that to allow ingest of files outside of the user's  home and default ingest directories, we do not need to add permissions to some other bundle too.

Reply:
Catalog core commands has access to all files.
https://github.com/codice/ddf/blob/cec2479856d5b9f2bcf8e3ff185679719cd98be6/distribution/ddf-common/src/main/resources/security/default.policy#L169-L171

any other bundles involved I suppose just happen to have the right permissions